docs/content/providers/kubernetes-crd.md

```diff
@@ -53,21 +52,34 @@ For this reason, users can run multiple instances of Traefik at the same time to
 
 When using a single instance of Traefik with LetsEncrypt, no issues should be encountered, however this could be a single point of failure.
```

Comment:
```suggestion
When using a single instance of Traefik with Let's Encrypt, you should encounter no issues. However, this could be a single point of failure.
```